enforcer-rules/README.md

```diff
@@ -0,0 +1,82 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule runs [Linkage Checker](../dependencies) for Maven projects.
+The rule verifies that the project does not have linkage errors in its class path.
+
+## Class path and bom flag
+
+The rule generates a class path (a list of jar files) from `pom.xml`.
+The generation logic differs for projects defining a BOM (BOM projects) and other projects.
+The rule takes `bom` flag to control the behavior (by default `false`).
+
+- When `bom=false`, the rule checks a class path consisting of the project's `dependencies` section
+  (immediate child element of `project`) and their transitive dependencies.
+- When `bom=true`, the rule checks a class path consisting of artifacts in the BOM's
+  `dependencyManagement` section and their transitive dependencies.
+
+# Usage
+
+Add the following plugin configuration to your `pom.xml`.
+
+```xml
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M2</version>
+        <dependencies>
+          <dependency>
+            <groupId>com.google.cloud.tools.opensource</groupId>
+            <artifactId>linkage-checker</artifactId>
```

Comment:
comparing with https://sonatype.github.io/ossindex-maven/enforcer-rules/ the artifact ID perhaps should be linkage-checker-enforcer-rules